lib/src/platform/src/realtime/realtime.dart

```diff
@@ -105,31 +108,48 @@ class Realtime extends PlatformObject {
   //   throw UnimplementedError();
   // }
 
-  /// gets stats based on params as a [PaginatedResult]
+  /// Queries the REST `/stats` API and retrieves your application's usage
```

Comment:
I think we could add an empty line after the first sentence to match https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Reply:
Done in 6c4b6b04df275b6136936cfec3d2b520b0477a55